README.md

```diff
@@ -146,33 +142,126 @@ There are two main steps in the pre-processing pipeline: creating the graph and
 
 The amount of pre-processing required will depend on what kind of datastore you will be using for training.
 
-### Additional inputs
-
-#### MultiZarr Datastore
+### Config
+When creating the graph for the model and for training, a configuration file is needed. The configuration file is a `.yaml` file that contains two dictionaries. One for `datastore` and one for `training`.
+An example of a configuration file for the DANRA dataset is given below:
+```yaml
+datastore:
+  kind: mdp
+  config_path: /path/to/mllam-data-prep-configuration.yaml
+training:
+  state_feature_weights:
+    pres_seasurface: 1.0
+    t2m: 1.0
+    r2m: 1.0
+    u10m: 1.0
+    v10m: 1.0
+```
+A reference to the configuration file used for mllam-data-prep is needed in the `datastore` dictionary. Please see the [mllam-data-prep](https://github.com/mllam/mllam-data-prep) repository for more information on how to create this configuration file.
+The `kind` key in the `datastore` dictionary specifies the type of datastore that will be used (should be one of `npyfiles`, or `mdp`).
 
-* `python -m neural_lam.create_boundary_mask`
-* `python -m neural_lam.create_datetime_forcings`
-* `python -m neural_lam.create_norm`
+### Additional inputs
 
 #### NpyFiles Datastore
 
 #### MDP (mllam-data-prep) Datastore
 
 An overview of how the different pre-processing steps, training and files depend on each other is given in this figure:
-<p align="middle">
-  <img src="figures/component_dependencies.png"/>
-</p>
-In order to start training models at least three pre-processing steps have to be run:
+```mermaid
+graph LR
+%% styling
+classDef proc_step stroke:#ffa49b
```

Comment:
Need to check if the mermaid diagram is up to date with the changes from the datastores work. Need input from @leifdenby here.

Reply:
I think the data loading part of that  diagram correspond to the NpyFilesDatastore, but does not match the zarr-based data loading in MDPDatastore. The graph-parts of it are still relevant, independent of the datastore. So maybe it would make more sense to split this up into multiple diagrams? E.g. one per datastore and one for the graph files?

Reply:
@khintz let's sit together today and redesign this. I think the README also needs an intro section that describes that we 1) use "data stores" to represent the input data to neural-lam and that 2) as well as the datastore configuration there are other *pre-processing* steps that must be done, specifically graph creation (and in addition the parameter weights for the npy-meps-based datastore)